### PR TITLE
Aperture theme - add smooth scrolling on click for hashed nav items

### PR DESCRIPTION
**Pull request by david-poindexter**
## Summary
This PR updates the new DNN 10 Aperture theme to have smooth scrolling for all hashed nav menu items.  The content that will be in the new `Default` template will have divs with ids for targeting those sections on the home page.

ApertureThemeSmoothScrolling

### CODE REVIEW

**Comment by david-poindexter** [comment]
I just pushed another commit to resolve an issue with smooth scrolling when logged into DNN.  The PR is now ready for review.

**Review by valadas — APPROVED** [review]
Awesome

File: DNN Platform/Skins/Aperture/src/scripts/menu.ts [hunk 1]
```diff
@@ -1,4 +1,31 @@
 export const navFunctions = (() => {
+	// add event listeners to all anchor tags with a hash in href
+	const anchorsWithHash: NodeListOf<HTMLAnchorElement> = document.querySelectorAll('a[href^="/#"]');
+	anchorsWithHash.forEach(anchorWithHash => {
+		// listen for click event
+		anchorWithHash.addEventListener('click', event => {
+			// prevent default anchor click behavior
+			event.preventDefault();
```
Comment (bdukes) [inline DNN Platform/Skins/Aperture/src/scripts/menu.ts:8]: we shouldn't prevent default if the link is on another page, right?

Comment (david-poindexter) [reply DNN Platform/Skins/Aperture/src/scripts/menu.ts]: @bdukes good catch - I just pushed a commit to resolve this.

**Review by bdukes — APPROVED** [review]

**Pull request merged**